[WIP]Ergänzungen in Log

GNCLORD-MDB opened the pull request:
Fehlen Informationen in den Logfiles #58

formtapez committed:
removed tools

formtapez committed:
removed build instructions from readme

senshi-x committed:
Merge pull request #45 from formtapez/master

removed tools

GNCLORD-MDB committed:
Merge pull request #4 from OperationPandoraTrigger/legacy

Abgleich

GNCLORD-MDB committed:
bessere Erfassung  der Munition/ Magazine

senshi-x commented:
Der Code sieht gut aus, aber an welcher Stelle ist er denn funktional anders? Also an welcher Stelle wird jetzt mehr erfasst als zuvor?

Bin vielleicht gerade noch zu blöd am Morgen, aber für mich sieht die Funktion gleich aus, nur hübscher geschrieben :D .

GNCLORD-MDB commented:
ja stimmt so gesehen die Funktion ist die gleiche. 
Muss dann nochmal schauen wie so es seine kann, das viele unbekannte kills in den Logs sind.

Frozen-byte commented:
Ist dieser PR noch WIP? 
Welche Änderungen gibt es? ("_einige Daten_" sind etwas schwammig)

GNCLORD-MDB commented:
ja, ich habe noch nicht herausgefunden warum viele unbekannt kill logs und logs ohne Waffentyp gibt.

GNCLORD-MDB committed:
Merge pull request #5 from OperationPandoraTrigger/master

update

GNCLORD-MDB committed:
Merge pull request #6 from OperationPandoraTrigger/legacy

Update

GNCLORD-MDB committed:
Dialoge erstellt

Frozen-byte committed:
initial arma thunder commit

GNCLORD-MDB committed:
OPT Mission Tanoa

GNCLORD-MDB committed:
Merge pull request #7 from GNCLORD-MDB/Legacy

Legacy

Frozen-byte committed:
Tanoa changes witout moving files

Frozen-byte committed:
rename folder for correct Map identification

Commit:
Changes for Prolog Schlacht

- Added Tempest Medical to Beamsystem
- Added Flag Positions for Prolog Schlacht
- Defined Mission Name
- Added Beampositions for Prolog Schlacht

Frozen-byte committed:
Merge pull request #95 from Yayshan/tanoa_2020H1

Added Tempest Medical to Beamsystem
Added Flag Positions for Prolog Schlacht
Defined Mission Name
Added Beampositions for Prolog Schlacht

Commit:
add loadouts.hpp

formtapez committed:
flaggenanzahl einstellmoeglichkeit

formtapez committed:
flaggenauswahlradius vergroessert

formtapez committed:
multi-auswahl auf der karte

formtapez committed:
marker nach missionsstart und failsafe-fill

Commit:
Update Mission for War

- Removed FPS Hungry Objects
- TFAR fix
- added trigger for vehicle delte on bridges

Commit:
Merge pull request #103 from Yayshan/tanoa_2020H1

Update Mission for War

formtapez committed:
fixed dominator calculation

formtapez committed:
fixed flagmarker deletion

formtapez committed:
fixed flag-moving (ALT-method)

formtapez committed:
rewrite of flag-moving (DIALOG-method)

formtapez committed:
cba-setting default flags and dupe removal

formtapez committed:
precise score counting with second resolution

formtapez committed:
fixed FreeMineZoneMarker names

formtapez committed:
fixed scoring

formtapez committed:
minezonenmarker now visibler and moving with flag

formtapez committed:
fixed opt-flag-hotkey

Commit:
Update Mission for War

Commit:
delete unnecessary file

Commit:
Merge pull request #104 from Yayshan/tanoa_2020H1

Update Mission  for next War

formtapez committed:
async flag distribution, optional neutral start

formtapez committed:
emergency exit for while-loop

formtapez committed:
fixed opt-marker bug

formtapez committed:
solved merge conflict

formtapez committed:
forEach -> apply

formtapez committed:
improved maxTries readability

formtapez committed:
fixed maxTries

formtapez committed:
reverted array madness

formtapez committed:
updated mission to upstream state

formtapez committed:
flagchooser overhaul

formtapez committed:
trying to fix some merge conflicts

formtapez committed:
fix conflicts

formtapez committed:
verflixter conflict fix

formtapez committed:
Krzmbrzlistic performance optimizations

Frozen-byte committed:
Merge pull request #101 from formtapez/spass_mit_flaggen

Flaggenscript verbessert.
Sekundengenaue Anrechnung der Punkte
N Fahnen möglich
eine Fahne kann nun neutral sein

formtapez committed:
imported oppas schlacht 3

Frozen-byte committed:
Merge pull request #108 from formtapez/tanoa_2020H1

schlacht 3

formtapez committed:
merge oppa version 107T_1

Frozen-byte committed:
Merge pull request #109 from formtapez/tanoa_2020H1

merge oppa version 107T_1

formtapez committed:
added action to show own position

formtapez committed:
added action for officer to edit beam locations

formtapez committed:
location limitation and logging

formtapez committed:
requested changes

formtapez committed:
dialog redesign

Frozen-byte committed:
Merge pull request #111 from formtapez/beam_overhaul

Beampunkt-Selbstverwaltung
* Mausrad-Action für Jeden um im Training die eigene Position als Vorlage für Beampunkte herauskopieren zu können
* Mausrad-Action für Offiziere um vor Missionsstart die Beampunkte festzulegen
* Beampunkt-Anzahl limitieren
* Beampunkte bei Missionsstart loggen

GNCLORD-MDB committed:
leeren

GNCLORD-MDB committed:
abgleichen

GNCLORD-MDB committed:
Merge branch

GNCLORD-MDB committed:
abgleich

GNCLORD-MDB committed:
Merge branch 'tanoa_2020H1'

Frozen-byte commented:
obsolete
